### PR TITLE
Add metric for latest block timestamp

### PR DESCRIPTION
**Pull request by Legogris**

### CODE REVIEW

File: internal/collector/eth_block_timestamp.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+func (collector *EthBlockTimestamp) Collect(ch chan<- prometheus.Metric) {
+	var bn hexutil.Uint64
+	var result *blockResult
+
+	if err := collector.rpc.Call(&bn, "eth_blockNumber"); err != nil {
+		ch <- prometheus.NewInvalidMetric(collector.desc, err)
+		return
+	}
+
+	if err := collector.rpc.Call(&result, "eth_getBlockByNumber", bn, 0); err != nil {
```
Comment (31z4) [inline internal/collector/eth_block_timestamp.go:45]: I suggest using `latest` as a parameter to that call. You'll get the latest block and do not have to call `eth_blockNumber`. See https://openethereum.github.io/JSONRPC-eth-module#eth_getblockbynumber for details.

File: internal/collector/eth_block_timestamp.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+type blockResult struct {
+	Timestamp hexutil.Uint64
+}
+
+func NewEthBlockTimestamp(rpc *rpc.Client) *EthBlockTimestamp {
+	return &EthBlockTimestamp{
+		rpc: rpc,
+		desc: prometheus.NewDesc(
+			"eth_block_timestamp",
```
Comment (31z4) [inline internal/collector/eth_block_timestamp.go:24]: Please also update https://github.com/31z4/ethereum-prometheus-exporter/blob/master/README.md#exported-metrics with metric's name and description.

File: internal/collector/eth_block_timestamp.go (hunk 1 above)
Comment (Legogris) [reply internal/collector/eth_block_timestamp.go]: Great, I missed that one! Updated.

**Comment by Legogris** [comment]
@31z4 thanks for the feedback, updated!

**Comment by 31z4** [comment]
Closing in favour of #16.

**Pull request closed without merging**